src/main/java/de/tum/cit/aet/artemis/atlas/api/AbstractAtlasApi.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.cit.aet.artemis.atlas.api;
+
+import de.tum.cit.aet.artemis.core.api.AbstractApi;
+
+public abstract class AbstractAtlasApi implements AbstractApi {
```

Comment:
```suggestion

@Controller
@Profile(PROFILE_CORE)
public abstract class AbstractAtlasApi implements AbstractApi {
```
Can we move the annotations of the subclasses here? This would also make it easier to introduce a new profile for Atlas later on

Reply:
No, Spring unfortunately does not support this.